en/mkdocs.yml

```diff
@@ -84,8 +84,10 @@ nav:
     - 'API Security':
        - 'Introduction': learn/api-security.md
        - 'Configuration': learn/api-security-configuration.md
+    - 'Administration API': learn/admin-api.md
```

Comment:
Suggestion:

- Learn section should contain a bried intro to the feature. 
- The content in 'en/docs/learn/admin-api.md' should be added under the 'try out' section